dotCMS/src/integration-test/java/com/dotcms/publishing/job/TestCaseSiteSearchBuilder.java

```diff
@@ -0,0 +1,107 @@
+package com.dotcms.publishing.job;
+
+import com.dotcms.publishing.job.SiteSearchJobImplTest.TestCaseSiteSearch;
+
+public class TestCaseSiteSearchBuilder {
+
+    private boolean defaultPageToDefaultLanguage;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field defaultPageToDefaultLanguage has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5303548)